src/components/AssetManagementPanel/AssetManagementForm/FormVariants/EditableAssetTitle/EditableAssetTitle.tsx

```diff
@@ -10,9 +10,17 @@ interface EditableAssetTitleProps {
   isEditable: boolean;
   newValue?: string;
   onSetNewValue?: (newBeneficiary: string) => void;
+  errorState?: string;
```

Comment:
```suggestion
  changeHolderState?: string;
```

Reply:
`error` or `errorMessage` will make more sense in this case

Reply:
ok